json-lint/schemas/inventory/adservers/adthrive.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$id": "#AdThrive",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "AdThrive adserver",
+  "description": "Ad server not supported by amp. Only Web",
+  "additionalProperties": false,
+  "required": [
+    "type",
+    "id",
+    "location",
+    "sticky"
+  ],
+  "defaultSnippets": [
+    {
+      "label": "AdThrive template",
+      "description": "contains required keys",
+      "body": {
+        "type": "adThrive",
+        "id": "$1",
+        "location": "$2",
+        "sticky": "$3"
+      }
+    }
+  ],
+  "properties": {
+    "type": {
+      "const": "adThrive"
+    },
+    "id": {
+      "type": "string"
```

Comment:
Ehy @rafaguerrero, the idea we had with web-only adserver is that we put all the non amp parameters in a `json` pocket just like smartadserver. This way if by mistake you place it in an amp position it will not trigger an invalid page. We also have nice-to-have task with validation of web only placement for any adserver that is marked as web. What do you think?